generator/src/main/java/com/scottlogic/deg/generator/outputs/manifest/ManifestWriterImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.scottlogic.deg.generator.outputs.manifest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.scottlogic.deg.generator.utils.FileUtils;
+import com.scottlogic.deg.generator.violations.ViolatedProfile;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.text.DecimalFormat;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+
+public class ManifestWriterImpl implements ManifestWriter{
```

Comment:
Could this be renamed to clarify what it does; e.g. `JsonManifestWriter` or something to this effect, then the `Impl` suffix can be dropped.

Reply:
yep. done.

Reply:
done